apps/frontend/app/routes/_dashboard.media.$action.$lot.tsx

```diff
@@ -135,6 +136,7 @@ export const loader = unstable_defineLoader(async ({ request, params }) => {
 					.nativeEnum(MediaGeneralFilter)
 					.default(defaultFilters.mineGeneralFilter),
 				collection: z.string().optional(),
+				invertCollection: z.string().optional(),
```

Comment:
make this `zx.BoolAsString.optional()`

Reply:
Ill make this change and add it to the people and groups page tomorrow thanks!